styles.css

```diff
@@ -35,6 +35,7 @@ body {
 .controls > button:hover {
   background: #e07999;
   color: white;
+  cursor: pointer;
```

Comment:
It's usually a good idea to add `cursor: pointer;` to hover effects on all buttons